.github/workflows/sync-helpz.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Sync mirrorz-help-ng'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '33 22 * * *'  # 06:33 AM in Asia/Shanghai (UTC+8)
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        submodules: true
+    - name: Sync submodule
+      run: |
+        git submodule update --remote helpz/_posts
+        echo -n 'See content changes in: ' > /tmp/pr_msg
+        echo -n 'https://github.com/tuna/mirrorz-help-ng/compare/' >> /tmp/pr_msg
```

Comment:
Maybe fetching submodule hash before and after running `submodule update` would be neater?

Reply:
Done.